portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>Thnx for Coming!</h1>
+      <div class="center">
+    <center>
+        <img src="images/IMG_0006.JPG"  style = "width:100px height:200px"/></a> <br>
```

Comment:
Rather than having the style defined directly in the html, once you figure out how you want it styled, add that to your CSS file and put the tag here. (Totally okay to do that in a future PR if you think you are still fiddling with the styling of these img)

Reply:
probably will do in future because the images have different sizes 
I'll play around with maybe a handful of classes ! thanks for the suggestion